.github/workflows/build.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       run: dotnet build --configuration Release
 
     - name: Publish the project
-      run: dotnet publish -c Release -o out
+      run: dotnet publish -c Release -r win-x64 --self-contained -p:PublishSingleFile=true -o out
```

Comment:
We should probably specify the project (RacingAidWpf) to publish.
I think this would mean it would look like `dotnet publish ./RacingAidWpf/RacingAidWpf.csproj -c Release -r win-x64 --self-contained -p:PublishSingleFile=true -o out`